src/utils/createUserCreationRequest.ts

```diff
@@ -0,0 +1,12 @@
+import { translit } from '@taskany/bricks';
+
+interface LoginAutoType {
+    firstName?: string;
+    middleName?: string;
+    surname?: string;
+}
+
+export const loginAuto = ({ firstName, middleName, surname }: LoginAutoType) => {
+    const login = (firstName?.slice(0, 1) || '') + (middleName?.slice(0, 1) || '') + surname;
```

Comment:
Fallback for optional surname prop

Otherwise concatenate will returns `undefined` as string value

Reply:
made surname = '' by default on 9 string